### PR TITLE
Add landing page

### LINKED ISSUE #19: Add landing page
The current [landing page](https://experiment-metadb-seven.vercel.app/) is blank. 

- [x] Add landing page with links to the [experiments DB](https://experiment-metadb-seven.vercel.app/catelog) and [release-builds DB](https://experiment-metadb-seven.vercel.app/release-provenance)

### PR DESCRIPTION
**Pull request by atteggiani**
Fixes #19

### CODE REVIEW

**Comment by aidanheerdegen** [comment]
Looks good to me. I've pinged Tommy for a review too.

File: frontend/src/app/page.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+      <div className={classes.logo}>
+        <Image src="/ACCESS-NRI_logo.png" alt="ACCESS-NRI_logo" fill />
+      </div>
+      <div className={`${classes.title} ${classes.dongle}`}>
+        ACCESS-NRI Data Explorer
+      </div>
+    </div>
+    <div className={`${classes.cardContainer} ${classes.dongle}`}>
+      <Link href="/catelog" className={classes.card}>Experiments Catalog</Link>
+      <Link href="/release-provenance" className={classes.card}>ACCESS-NRI Releases Provenance</Link>
```
Comment (aidanheerdegen) [inline frontend/src/app/page.js]:
```suggestion
      <Link href="/release-provenance" className={classes.card}>Model Build Database</Link>
```
Is the ACCESS-NRI implied? I think I'd slightly prefer the suggested name as it is a bit more specific. The Build DB is a crucial part of the Release Provenance, but only a part.

**Review by aidanheerdegen — APPROVED** [review]
Awesome, thanks very much.

File: frontend/src/app/page.js (hunk 1 above)
Comment (atteggiani) [reply frontend/src/app/page.js]: Sure did not know exactly what names I should give, so I just tried to infer from the content :)
Committed your suggestion!

**Review by CodeGat — APPROVED** [review]
Lovely, although my web dev skills are limited

**Review by utkarshgupta95 — APPROVED** [review]

File: frontend/src/app/page.js [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
+      </div>
+      <div className={`${classes.title} ${classes.dongle}`}>
+        ACCESS-NRI Data Explorer
+      </div>
+    </div>
+    <div className={`${classes.cardContainer} ${classes.dongle}`}>
+      <Link href="/catelog" className={classes.card}>Experiments Catalog</Link>
+      <Link href="/release-provenance" className={classes.card}>Model Build Database</Link>
+    </div>
+  </>
```
Comment (aidanheerdegen) [inline frontend/src/app/page.js]: Can we swap the ordering of these, and add “(experimental prototype)” to the Data Explorer. I would like to indicate it is not necessarily intended to be a final product.

Comment (atteggiani) [reply frontend/src/app/page.js]: Done.

**Pull request merged**